### ISSUE TITLE
Using TailwindCSS with `deploySite` results in `RangeError: Maximum call stack size exceeded`

### ISSUE THREAD

**Issue by Just-Moh-it** [remotion-dev/remotion#1901]
# Bug Report 🐛

After adding TailwindCSS to an existing project (using the way the docs suggest), and adding the `enable-tailwind.ts` file to the `deploySite` function in the deploy script, it results in this error:

```shell
RangeError: Maximum call stack size exceeded
    at RegExp.exec (<anonymous>)
    at retrieveSourceMapURL (/Users/mohit/.npm/_npx/1bf7c3c15bf47d04/node_modules/@cspotcode/source-map-support/source-map-support.js:340:21)
    at Array.<anonymous> (/Users/mohit/.npm/_npx/1bf7c3c15bf47d04/node_modules/@cspotcode/source-map-support/source-map-support.js:353:26)
    at /Users/mohit/.npm/_npx/1bf7c3c15bf47d04/node_modules/@cspotcode/source-map-support/source-map-support.js:199:32
    at mapSourcePosition (/Users/mohit/.npm/_npx/1bf7c3c15bf47d04/node_modules/@cspotcode/source-map-support/source-map-support.js:383:21)
    at wrapCallSite (/Users/mohit/.npm/_npx/1bf7c3c15bf47d04/node_modules/@cspotcode/source-map-support/source-map-support.js:592:20)
    at Function.prepareStackTrace (/Users/mohit/.npm/_npx/1bf7c3c15bf47d04/node_modules/@cspotcode/source-map-support/source-map-support.js:671:41)
    at maybeOverridePrepareStackTrace (node:internal/errors:142:29)
    at prepareStackTrace (node:internal/errors:116:5)
    at errorStack (/Users/mohit/dev/maxroom/maxroom-v2/node_modules/.pnpm/webpack@5.74.0_esbuild@0.16.12/node_modules/webpack/lib/stats/DefaultStatsFactoryPlugin.js:455:25
```

However, removing the TailwindCSS override still results in an error.

## Steps to reproduce 🤰

1. Go to the URL: https://stackblitz.com/edit/github-anmufe
2. Download the project locally through the download icon on stackblitz (because websockets used for deployment have a hard time running in stackblitz 🙃)
3. On your local instance, npm install and npm start to verify everything is working
4. Add your AWS access keys and secrets in the `.env` file
5. Run the `deploy` script in the `package.json` to see the error
6. To get rid of the error, comment out the line `49` in `deploy.ts`, or remove the webpack override from `deploySite`. Though this would remove TailwindCSS support

## Environment 🌿

I'm somewhat sure this problem only occurred when I upgraded from Remotion `3.3.32` to `3.3.60`

Been a big problem across various projects I've been working on, cc @JonnyBurger

**Comment by Tonie-NG** [comment]
@Just-Moh-it  Could you provide more clarity on this issue?

**Comment by JonnyBurger** [comment]
I think i can repro it, will take a look at it today

**Comment by Tonie-NG** [comment]
Would love to hear your thoughts on it when you do @JonnyBurger

**Comment by JonnyBurger** [comment]
I followed the instructions but actually I could not reproduce!

Everything deployed correctly even the functions:

![image](https://user-images.githubusercontent.com/1629785/223465124-019547d5-dd3f-43c4-8850-66ea471d38e8.png)

No lockfile changes either. Nonetheless I have some ideas:
- The stacktrace says that this comes from @cspotcode/source-map-support. This is a dev dependency of ts-node. So it looks like  the curlprit is there
- Remotion bundler imports all of Webpack, maybe this is too heavy for ts-node. Try changing it to `ìmport type {WebpackOverrideFn} from "@remotion/bundler"` to not actually import all of it.

**Comment by Just-Moh-it** [comment]
Thanks for your suggestions, Jonny! Through 

> This is a dev dependency of ts-node

I was able to deduce that it might just be a problem with the ts-node runtime, so I asked my colleague to try it on their machine, and the script seemed to have worked. I moved from pnpx to npx and it worked too! But just the first time.

Finally, I had to install ts-node as a dev dependency of the project I was working on since it was the backbone for a huge part of the app (the deployment/deletion parts), and running just `ts-node` without npx/pnpx seems to not cause the issue!

Thanks for the suggestion in the second point, too!
